spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef SimplifiedLink
+ *
+ * @prop {string=} innerText
+ * @prop {string=} href
+ * @prop {string=} className
+ * @prop {string=} target
```

Comment:
I don't think any of these would ever be undefined?

```suggestion
 * @prop {string} innerText
 * @prop {string} href
 * @prop {string} className
 * @prop {string} target
```

As an alternative, if we expect some properties could be absent, we can use the `Partial<SimplifiedLink>` type where we expect that.

https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype

Reply:
I guess technically `target` could be undefined, but I see your point

Reply:
> I guess technically `target` could be undefined, but I see your point

From what I can tell, even if unassigned, `target` will be a string.

https://github.com/microsoft/TypeScript/blob/d36df0d/src/lib/dom.generated.d.ts#L6182-L6185

On that note, there could be an option here to select a subset of TypeScript's DOM definition for an anchor tag, like:

```js
/** @typedef {Pick<HTMLAnchorElement, "innerText" | "href" | "className" | "target">} */
```

https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
Neat! Added it in 2152198a9e6c732c8d9f41ee16cc4586d0e79cfd